move/its/sources/its.move

```diff
@@ -270,6 +278,22 @@ public fun set_flow_limit<T>(
     );
 }
 
+// This is the entrypoint for operators to set the flow limits of their tokens
+// (interchainTokenService.setFlowLimits on EVM)
+public fun set_flow_limit_as_operator<T>(
```

Comment:
this should be the default `set_flow_limit` method. The other method can be renamed to `set_flow_limit_by_token_distributor`

Reply:
use `set_flow_limit_as_token_operator` instead